loader/0002-quilt.mod.json.md

```diff
@@ -0,0 +1,362 @@
+# Summary
+This document acts as a Request for Comments (RFC) on the topic of how a Quilt mod's metadata should be formatted. Such changes are intended to provide added value for mod developers, library developers, and users alike.
+
+# Motivation
+Since the toolchain is being forked, now is the opportune moment to make large scale changes to many of the systems mod developers use. This RFC describes a more flexible and powerful format for the mod file that defines what a mods definition should look like.
+
+# Explanation
+Below is an outline of all defined keys and values.
+
+* [schema_version](#the-schema_version-field) — The schemaVersion to be used for reading this file
+* [quilt_loader](#the-quilt_loader-field) — Information related to loading the mod
+    * [group_id](#the-group_id-field) — The Maven group id
+    * [mod_id](#the-mod_id-field) — The mod id
+    * [provides](#the-provides-field) — Alternative mod ids provided by this mod
+    * [version](#the-version-field) — The mods version
+    * [entrypoints](#the-entrypoints-field) — Collection of entrypoints
+    * [plugins](#the-plugins-field) — Collection of plugins
+    * [jars](#the-jars-field) — Array of nested JARs to be loaded
+    * [language_adapters](#the-language_adapters-field) — Array of language adapters
+    * [depends](#the-depends-field) — Collection of mod dependencies
+    * [breaks](#the-breaks-field) — Collection of mods that this mod is incompatible with
+    * [repositories](#the-repositories-field) — Array of maven repositories
+    * [metadata](#the-metadata-field) — Extra information about this mod and/or its authors
+        * [name](#the-name-field) — Human-readable name of this mod
+        * [description](#the-description-field) — Human-readable description of this mod
+        * [contributors](#the-contributors-field) — Collection of contributors to this mod
+        * [contact](#the-contact-field) — Collection of contact information
+        * [license](#the-license-field) — One or more licenses this project is under
+        * [icon](#the-icon-field) — The icon or icons associated with this project
+* [mixin](#the-mixin-field) — Path(s) to mixin config file(s)
+* [access_widener](#the-access_widener-field) — Path(s) to accesswidener file(s)
+* [minecraft](#the-minecraft-field) - Minecraft related options
+    * [environment](#the-environment-field) — What game environments this mod applies to
+    
+## The `schema_version` field
+| Type   | Required |
+|--------|----------|
+| Number | True     |
+
+The quilt mod file schema version to be used for parsing this file. Currently, the only valid version is 1.
```

Comment:
Why is the first schema version `1` instead of `0`?

Reply:
Version 0 in a way implies unreleased.